### PR TITLE
create Sentinel API .Net core sample

### PR DESCRIPTION
**Pull request by chinguyen1**

### CODE REVIEW

**Comment by chinguyen1** [comment]
@preetikr I closed the original PR: https://github.com/Azure/Azure-Sentinel/pull/996 & opened this new one with some additions and fixes. 
Kobi's team (Actions API) has reviewed his part of API & all looks good. I'm still waiting on other teams to review their parts. But these parts are essentially similar to Actions API. If you're ok with it, we can merge this, then if users have any questions, they can raise issues & I will help take care of those.

**Review by preetikr — APPROVED** [review]

**Pull request merged**
